packages/custom-storage/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_SERVER = 'https://{user}.carto.com/';
+export const DEFAULT_API_KEY = 'default_public';
```

Comment:
Yeah, that 'common' again

Reply:
Just tested this, looks like you can have a common folder you can import, so we could put some constants, utils and classes in there.

Reply:
Actually, no, all source code must have the same common top folder.

Reply:
That's cool! We can talk about it tomorrow.

Reply:
Let's keep the conversation outside of this PR --> https://github.com/CartoDB/toolkit/issues/17